components/library/jxrlib/patches/01-Makefile.patch

```diff
@@ -0,0 +1,56 @@
+ 
+ $(DIR_BUILD)/libjxr.pc: $(DIR_SRC)/libjxr.pc.in
+-	@python -c 'import os; d = { "DIR_INSTALL": "$(DIR_INSTALL)", "JXR_VERSION": "$(JXR_VERSION)", "JXR_ENDIAN": "$(ENDIANFLAG)" }; fin = open("$<", "r"); fout = open("$@", "w+"); fout.writelines( [ l % d for l in fin.readlines()])'
++	@python -c 'import os; d = { "PREFIX": "$(PREFIX)", "JXR_VERSION": "$(JXR_VERSION)", "JXR_ENDIAN": "$(ENDIANFLAG)" }; fin = open("$<", "r"); fout = open("$@", "w+"); fout.writelines( [ l % d for l in fin.readlines()])'
+ 
+ install: all $(DIR_BUILD)/libjxr.pc
+-	install -d $(DIR_INSTALL)/lib/pkgconfig $(DIR_INSTALL)/bin $(DIR_INSTALL)/include/libjxr/common  $(DIR_INSTALL)/include/libjxr/image/x86 $(DIR_INSTALL)/include/libjxr/glue $(DIR_INSTALL)/include/libjxr/test $(DIR_INSTALL)/share/doc/jxr-$(JXR_VERSION)
+-	install $(LIBRARIES) $(DIR_INSTALL)/lib
+-	install -m 644 $(DIR_BUILD)/libjxr.pc $(DIR_INSTALL)/lib/pkgconfig
+-	install $(DIR_BUILD)/$(ENCAPP) $(DIR_BUILD)/$(DECAPP) $(DIR_INSTALL)/bin
+-	install -m 644 $(DIR_SRC)/common/include/*.h $(DIR_INSTALL)/include/libjxr/common
+-	install -m 644 $(DIR_SRC)/image/x86/*.h $(DIR_INSTALL)/include/libjxr/image/x86
+-	install -m 644 $(DIR_SRC)/$(DIR_SYS)/*.h $(DIR_INSTALL)/include/libjxr/image
+-	install -m 644 $(DIR_SRC)/$(DIR_ENC)/*.h $(DIR_INSTALL)/include/libjxr/image
+-	install -m 644 $(DIR_SRC)/$(DIR_DEC)/*.h $(DIR_INSTALL)/include/libjxr/image
+-	install -m 644 $(DIR_SRC)/$(DIR_GLUE)/*.h $(DIR_INSTALL)/include/libjxr/glue
+-	install -m 644 $(DIR_SRC)/$(DIR_TEST)/*.h $(DIR_INSTALL)/include/libjxr/test
+-	install -m 644 doc/* $(DIR_INSTALL)/share/doc/jxr-$(JXR_VERSION)
++	ginstall -d $(DIR_INSTALL)/usr/lib/amd64/pkgconfig $(DIR_INSTALL)/usr/bin $(DIR_INSTALL)/usr/include/libjxr/common  $(DIR_INSTALL)/usr/include/libjxr/image/x86 $(DIR_INSTALL)/usr/include/libjxr/glue $(DIR_INSTALL)/usr/include/libjxr/test $(DIR_INSTALL)/usr/share/doc/jxr-$(JXR_VERSION)
++	ginstall $(LIBRARIES) $(DIR_INSTALL)/usr/lib/amd64/
++	ginstall -m 644 $(DIR_BUILD)/libjxr.pc $(DIR_INSTALL)/usr/lib/amd64/pkgconfig
++	ginstall $(DIR_BUILD)/$(ENCAPP) $(DIR_BUILD)/$(DECAPP) $(DIR_INSTALL)/usr/bin
++	ginstall -m 644 $(DIR_SRC)/common/include/*.h $(DIR_INSTALL)/usr/include/libjxr/common
++	ginstall -m 644 $(DIR_SRC)/image/x86/*.h $(DIR_INSTALL)/usr/include/libjxr/image/x86
++	ginstall -m 644 $(DIR_SRC)/$(DIR_SYS)/*.h $(DIR_INSTALL)/usr/include/libjxr/image
++	ginstall -m 644 $(DIR_SRC)/$(DIR_ENC)/*.h $(DIR_INSTALL)/usr/include/libjxr/image
++	ginstall -m 644 $(DIR_SRC)/$(DIR_DEC)/*.h $(DIR_INSTALL)/usr/include/libjxr/image
++	ginstall -m 644 $(DIR_SRC)/$(DIR_GLUE)/*.h $(DIR_INSTALL)/usr/include/libjxr/glue
++	ginstall -m 644 $(DIR_SRC)/$(DIR_TEST)/*.h $(DIR_INSTALL)/usr/include/libjxr/test
++	ginstall -m 644 doc/* $(DIR_INSTALL)/usr/share/doc/jxr-$(JXR_VERSION)
+ 
```

Comment:
Maybe you should try to use
PATH=$(PATH.gnu)
in the Makefile. It should let the build system find the gnu install version.